src/grocker/six.py

```diff
@@ -40,3 +41,15 @@ def __enter__(self):
 
     def __exit__(self, exc, _value, _tb):
         shutil.rmtree(self.name)
+
+
+def getstatusoutput(cmd):
+    try:
+        data = subprocess.check_output(cmd, shell=True, universal_newlines=True, stderr=subprocess.STDOUT)
+        exitcode = 0
+    except subprocess.CalledProcessError as ex:
+        data = ex.output
+        exitcode = ex.returncode
+    if data[-1:] == '\n':
```

Comment:
`data = subprocess.check_output(cmd, universal_newlines=True, stderr=subprocess.STDOUT).strip()` should do the trick

Reply:
It's a duplication of https://github.com/python/cpython/blob/3.6/Lib/subprocess.py#L495 for compatibility purpose (Python 2.7). But according to the documentation it's a legacy function, maybe I should replace it by an another subprocess function.